cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/AbstractConductingEquipmentConversion.java

```diff
@@ -219,18 +219,32 @@ public PowerFlow terminalPowerFlow(int n) {
         return terminals[n - 1].t.flow();
     }
 
-    PowerFlow equipmentPowerFlow() {
-        return equipmentPowerFlow;
+    PowerFlow steadyStatePowerFlow() {
+        return steadyStatePowerFlow;
     }
 
     PowerFlow powerFlow() {
-        // Could come either from terminal data or from property bag
-        return terminalPowerFlow().defined() ? terminalPowerFlow() : equipmentPowerFlow();
+        // used for state attributes (targetQ, targetP, p0, etc.) outside boundaries
+        // SSH files are priority
+        if (steadyStatePowerFlow().defined()) {
```

Comment:
Needs rethinking.

We are changing the priority for all types of equipment.

The initial problem was raised because for energy consumers `(pfixed, qfixed)` from EQ profile was taken first. 

Maybe we should analyze all the different scenarios (energy consumers, injections from generators, ...) and make explicit decisions instead of applying a generic approach.

In general, if we want to "copy" on IIDM the power flow solution from CGMES, we should preserve the SV values first. If not available, then SSH. As a last resort, attributes that could come from EQ definition.

For example, for generators regulating voltage, if we copy the `q`value from the SV we will be able to observe potential problems in the solution given. If we copy the `q` from the SSH we won't be able to analyze in IIDM it the solution was "right" or "wrong".